recipes-graphics/userland/userland/0017-khronos-backport-typedef-for-EGL_EXT_image_dma_buf_i.patch

```diff
@@ -0,0 +1,22 @@
+From ffc50f94bd172d02042d334464eb911c1c4ff47f Mon Sep 17 00:00:00 2001
+From: Hugo Hromic <hhromic@gmail.com>
+Date: Sun, 13 May 2018 10:49:04 +0100
+Subject: [PATCH] khronos: backport typedef for EGL_EXT_image_dma_buf_import
+
```

Comment:
Please include an upstream status.

Reply:
My first own patch submitted, sorry I didn't read the guideline for formatting. Fixed now!